www/include/monitoring/status/HostGroups/xml/hostGroupXML.php

```diff
@@ -50,91 +50,98 @@
     exit();
 }
 
-/*
- * Set Default Poller
- */
+// Set Default Poller
 $obj->getDefaultFilters();
 
-/*
- * Alias / Name conversion table
- */
-$convertTable = array();
-$convertID = array();
-$DBRESULT = $obj->DBC->query("SELECT hostgroup_id, name FROM hostgroups");
-while ($hg = $DBRESULT->fetchRow()) {
+// Alias / Name conversion table
+$convertTable = [];
+$convertID = [];
+$dbResult = $obj->DBC->query("SELECT hostgroup_id, name FROM hostgroups");
+while ($hg = $dbResult->fetch()) {
     $convertTable[$hg["name"]] = $hg["name"];
     $convertID[$hg["name"]] = $hg["hostgroup_id"];
 }
-$DBRESULT->closeCursor();
+$dbResult->closeCursor();
 
 // Check Arguments From GET tab
-$o = filter_input(INPUT_GET, 'o', FILTER_SANITIZE_STRING, array('options' => array('default' => 'h')));
-$p = filter_input(INPUT_GET, 'p', FILTER_VALIDATE_INT, array('options' => array('default' => 2)));
-$num = filter_input(INPUT_GET, 'num', FILTER_VALIDATE_INT, array('options' => array('default' => 0)));
-$limit = filter_input(INPUT_GET, 'limit', FILTER_VALIDATE_INT, array('options' => array('default' => 20)));
+$o = filter_input(INPUT_GET, 'o', FILTER_SANITIZE_STRING, ['options' => ['default' => 'h']]);
+$p = filter_input(INPUT_GET, 'p', FILTER_VALIDATE_INT, ['options' => ['default' => 2]]);
+$num = filter_input(INPUT_GET, 'num', FILTER_VALIDATE_INT, ['options' => ['default' => 0]]);
+$limit = filter_input(INPUT_GET, 'limit', FILTER_VALIDATE_INT, ['options' => ['default' => 20]]);
 //if instance value is not set, displaying all active pollers linked resources
 $instance = filter_var($obj->defaultPoller ?? -1, FILTER_VALIDATE_INT);
 
-$search = filter_input(INPUT_GET, 'search', FILTER_SANITIZE_STRING, array('options' => array('default' => '')));
-$sort_type = filter_input(
```

Comment:
in this list, we can only sort one column. This element is not needed nor usefull currently